common/remote_object.go

```diff
@@ -145,7 +186,7 @@ func handleParseRemoteObjectErr(ctx context.Context, err error, logger *logrus.E
 		ooe *objectOverflowError
 		ope *objectPropertyParseError
 	)
-	merr, ok := err.(*multierror.Error)
+	merr, ok := err.(*multiError)
```

Comment:
The problem with `As()` AFAIK is that it will return the first error in the chain that matches the given type, if found any. That means that you might skip some "wrapped errors" in the chain, but considering the current implementation I think it should be safe here, changed [here](https://github.com/grafana/xk6-browser/blob/d76ff69916458824d567f80c8849a648b4efde8a/common/remote_object.go#L193).

Reply:
yep, makes sense. maybe add a `nolint` directive here and explain why.

Reply:
As mentioned, for our current implementation, using `As()` should be safe, changed.